BootloaderCommonPkg/Library/PartitionLib/PartitionLib.c

```diff
@@ -62,6 +63,25 @@ GetPartitionTypeName (
 }
 
 
+/**
+  Get the GPT entries array from the last GPT block
+  device read.
+
+  @param  None
+
+  @retval                   Pointer to the GPT entry data
+
+**/
+EFI_PARTITION_ENTRY *
+EFIAPI
+GetGptEntryData (
```

Comment:
Is anybody using this new API ?
And for what purpose ?

Reply:
Some Linux kernel command line parameters require GPT partition UUID information.

Reply:
Shouldn't the kernel command line parameter be provided by user or cfg file ?   Why do we need to know this in OsLoader ?
If this type of function is really required,  we can consider some API like  GetPartitionInfo() to abstract it.  But we still need to understand what data is necessary for bootloader to consume.